.gitignore

```diff
@@ -0,0 +1,7 @@
+.venv/
+venv/
+.idea/
+__pycache__/
+db.sqlite3
+.pyc
```

Comment:
.pyc won't exclude all .pyc files. If you want to exclude them all from all directories and subdirectories, use **/*.pyc